### PR TITLE
New version: Microsoft.PowerShell version 7.3.4.0

### PR DESCRIPTION
**Pull request by anamnavi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102538)

### CODE REVIEW

**Comment by mdanish-kh** [comment]
Resolves #102530

File: manifests/m/Microsoft/PowerShell/7.3.4.0/Microsoft.PowerShell.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- open-source
+- powershell
+- pwsh
+- shell
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/PowerShell/PowerShell/releases/tag/v7.3.4
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
```
Comment (mdanish-kh) [inline manifests/m/Microsoft/PowerShell/7.3.4.0/Microsoft.PowerShell.locale.en-US.yaml:35]:
```suggestion
Documentations:
- DocumentLabel: Product Documentation
  DocumentUrl: https://learn.microsoft.com/powershell
- DocumentLabel: FAQ
  DocumentUrl: https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md
```
Metadata to make the manifest more complete if you wish

**Review by mdanish-kh — APPROVED** [review]

**Comment by mdanish-kh** [comment]
cc @anamnavi Would suggest adding the optional Documentation field in the locale file to make the manifest more complete (I don't want to add it in a review comment as that blocks the PR merge process if you don't resolve the comment 🫠)

If the PR gets merged before that, would you mind if I add it in a separate PR?

```suggestion
Documentations:
- DocumentLabel: Product Documentation
  DocumentUrl: https://learn.microsoft.com/powershell
- DocumentLabel: FAQ
  DocumentUrl: https://github.com/PowerShell/PowerShell/blob/master/docs/FAQ.md
```

**Comment by BrandonWanHuanSheng** [comment]
That PR will not be merged due to Internal-Error-Dynamic-Scan.

**Comment by BrandonWanHuanSheng** [comment]
Oops! What it will be merged before the comments?

**Pull request merged**